src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -386,14 +414,15 @@ FlightContext resume(String stairwayId, String flightId) throws DatabaseOperatio
             return flightContext;
 
         } catch (SQLException ex) {
-            throw new DatabaseOperationException("Failed to get flight", ex);
+            handleSqlException("Failed to get flight", ex);
```

Comment:
Same comment as above, could you include the stairway and flight ids in the exception message?